PUBLISHING.md

```diff
@@ -8,21 +8,19 @@ procedure feels more comfortable. By publishing, the “Publish” GitHub Action
 workflow will be triggered and if all the tests pass, it’ll publish to all the
 registries we care about.
 
-Before you start — make sure your on the `main` branch and up to date. E.g., by
-running `git checkout main && git pull origin main`.
+Before you start — make sure you are on the `main` branch and up to date.
+E.g., by running `git checkout main && git pull origin main`.
 
 ## Publishing with Manual Prepublish
 
-1. Edit `package.json`, `package-lock.json`, and `deno.json` files to
-   set the new version. Don’t forget that `package-lock.json` should be edited
-   in multiple places to account for the self-package’s version (i.e., `""`).
-   For example, set the `"version"` key to a value of `"1.0.0-rc.57"`.
-2. Add / commit those files. By convention, set the message to the new version
-   (e.g., `git commit --message="1.0.0-rc.57"`).
-3. Tag the commit using an annotated tag. By convention, set the name to the new
-   version _prefixed with a “v”_ and set the message to the new version
-   (e.g., `git tag --annotate "v1.0.0-rc.57" --message="1.0.0-rc.57"`).
-4. Push the new commit / tags by running `git push origin main --follow-tags`.
+1. Edit `package.json` and `deno.json` files to set the new version.
+2. Run `npm install` to derive `package-lock.json` using the new version.
+2. Commit the changed files. Follow the convention for the commit message.
```

Comment:
Bump numbers here. We have two twos now.

Reply:
good catch! fixt.